little-books/src/components/pop-up-box/pop-up-box.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { blogActions } from '../../store/blog-store';
+import styles from './pop-up-box.module.scss';
+import Button from '../../components/button/button.tsx';
+import { stringConst } from '../../constants/string-constants';
+
+const PopUp = ()=>{
+
+    const popUpData = useSelector((state:any)=>state.popUp.updateBlog);
+    const theme = useSelector((state:any)=>state.theme);
+    const dispatch = useDispatch();
+
+    const onPopUpClose = ()=>{
+        dispatch(blogActions.popUpToggle(''))
+    };
+
+    const popUpOkFunc = ()=>{
+        dispatch(blogActions.updateSelectedBlog(popUpData));
+        dispatch(blogActions.editBlogStateToggle());
+        onPopUpClose();
+    }
+
+    return(
+        <>
+        <div className={styles['dark-bg']}/>
+            <div className={`${styles['pop-up-wrapper']} ${styles[theme]}`}>
+                <h5 className={styles['pop-up-header']}>{stringConst.popUpConst.HEADING}</h5>
+                <div className={styles['btn-section']}>
+                    <Button type='primary-btn' size='small' onClickHandler={popUpOkFunc}>{stringConst.popUpConst.YES_OPTION}</Button>
```

Comment:
strings and numbers can go into constants

Reply:
string moved to constants